DigitalLearningSolutions.Web/Controllers/MyAccountController.cs

```diff
@@ -88,22 +88,37 @@ public IActionResult EditDetails(EditDetailsViewModel model, string action)
 
         private IActionResult EditDetailsPostSave(EditDetailsViewModel model)
         {
-            ValidateCustomPrompts(model);
+            var userAdminId = User.GetAdminId();
+            var userDelegateId = User.GetNullableCandidateId();
 
-            if (!ModelState.IsValid)
+            if (userDelegateId.HasValue)
             {
-                return View(model);
+                ValidateJobGroup(model);
+                ValidateCustomPrompts(model);
             }
 
             if (model.ProfileImageFile != null)
             {
                 ModelState.AddModelError(nameof(EditDetailsViewModel.ProfileImageFile),
                     "Preview your new profile picture before saving");
-                return View(model);
             }
 
-            var userAdminId = User.GetAdminId();
-            var userDelegateId = User.GetNullableCandidateId();
+            var (adminUser, delegateUser) = userService.GetUsersById(userAdminId, userDelegateId);
+            if (adminUser?.EmailAddress != model.Email || delegateUser?.EmailAddress != model.Email)
+            {
+                var (adminUsersWithNewEmail, delegateUsersWithNewEmail) = userService.GetUsersByEmailAddress
+                    (model.Email!);
+                if (adminUsersWithNewEmail != null || delegateUsersWithNewEmail.Count != 0)
+                {
+                    ModelState.AddModelError(nameof(EditDetailsViewModel.Email),
+                        "A user with this email address already exists");

```

Comment:
Delegate emails are only unique per centre so you'll want to filter `delegateUsersWithNewEmail` by their centre id as well - you can see what I did in the HEEDLS-446 MR for inspiration